### ISSUE TITLE
Revision

### ISSUE THREAD

**Issue by apoorvagnihotri** [distillpub/post--bayesian-optimization#13]
- [x] Remove BO slides, they don't add value. e0eee6a35b741f75856eb9b4b9e16408a699eca7
- [x] Correct hyperlinks in Acknoledgement e88978cfb960289451e268e58d843ab706021ab3
- [x] Acknoledgement -> Acknoledgements e88978cfb960289451e268e58d843ab706021ab3
- [x] Content gets clipped in collapsible f28df304a8d77b6227b3b9c11e6cf056560a42ad
- [x] Writing Improvements e11591fd3655171269a8f3ecf9892c639459d867 723b2dc6982a158cbe62fc97db909393bd000971
  - [x] Include suggestions from Chris. 4b7055a1421864f367fa9123a9131bd18e79ea0f
  - [x] `Abstract` + `Mining Gold!` 1024eb51945b83b8e000ae7b98687c48620e46d4 91e2e1a32e9eb48ddecd842fea6024b829ccc679
  - [x] `Active Learning` 5539934c3cd56e134a67de16d194a4d8081f3a32
  - [x] `Bayesian Optimization` + `Formalizing Bayesian Optimization`  6e40a36d858e4100eb525c1e1572834737bc6663
  - [x] `Acquisition Functions` 4b6d52cf2d43a709ce83f270e76dbdcbb51aea24
  - [x] `Probability of Improvement (PI)` 3df731b316e06674dadb520e28fde13bedd1b450
  - [x] `Expected Improvement (EI)` 010cd004d9ac5e319eaf6c3d25faf38d0581c2e3
  - [x] `PI vs. EI` 87ac6d5434118ca8acdfe0fc57a63b0c791b92c1
  - [x] `Thompson Sampling` 5abe99b5c3d5f3137e1d677514c75d1d64d5da45
  - [x] `Random` + `Summary of Acquisition Functions`
  - [x] `Other Acq. Functions` 0be836f40ae1fd71daa53aa42c46beb04fed15ee
  - [x] `Comparison` + `Why is it beneficial to optimize the acquisition function?`
  - [x] `Hyperparameter Tuning` cb5b009751da86b3b5b2d95005726077b0e5cd77
  - [x] `Example 1, 2, 3` 629fa365fe66da3217f034d7463ba429ed37e368
  - [x] `Conclusion and Summary` + `Embrace Bayesian Optimization`
  - [x] `Acknowledgements`
  - [ ] `Further Reading`
- [x] make grids fainter 4cb44bea2f7bdfaf47f70c11402acb917d9cce55
- [x] In thompson acq. image, 6c8fd4633a77a6c53bb8bd6ccf03e3dfce8f7974
  - [x] Title -> "samples from surrogate posterior"
  - [x] Use black for Training Point.
  - [x] Translucent bounds
- [x] Interactive Plot 13f00c9aa34c665f2fcd96fb0de64cbb456189ee
  - [x] In Interactive plot, we do not need ground truth.
  - [x] New Interactive plot. 77a9c0c7adf9c713f2b6fc237719b0ffc55a30ba 1e6e1f41494bf85c0460b70907d467f56f7ae9ad
- [x] Summary BO Plot
- [x] Sliders don't span the whole image. 61eb725b540bf71efcd310c31ea572d2682500ef
- [x] In examples plots 3edb4f08592fbe796e3bbdb737f89161531a2883 7ba7bf85937e6b75c5442c7764fbb04a11ad224c
  - [x] lighter alpha across every plot 7ba7bf85937e6b75c5442c7764fbb04a11ad224c
  - [x] Iteration number should be common for the two subplots. (common title subplots `fig.text` used)
  - [x] replace pink dots with hollow circles with thick black borders.
  - [x] Have Text label with variable name.
    - [x] for alpha
    - [x] for C
    - [x] for gamma
